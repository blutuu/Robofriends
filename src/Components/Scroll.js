import React from 'react';

const Scroll = (props) => {
    return (
        <div className="ba bw2" style={{ height: '500px', overflow: 'scroll' }}> 
            {props.children}
        </div>
    );
}

export default Scroll;