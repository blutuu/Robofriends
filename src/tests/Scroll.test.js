import React from "react";
import { shallow } from "enzyme";
import Scroll from "../Components/Scroll";

let wrapper = shallow(<Scroll />);

describe("Scroll component testing", () => {
  it("renders the Scroll component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
