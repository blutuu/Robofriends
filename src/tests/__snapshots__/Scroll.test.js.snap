// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll component testing renders the Scroll component correctly 1`] = `
<div
  className="ba bw2"
  style={
    Object {
      "height": "500px",
      "overflow": "scroll",
    }
  }
/>
`;
